starwars_api/models.py

```diff
@@ -28,8 +33,11 @@ def all(cls):
         later in charge of performing requests to SWAPI for each of the
         pages while looping.
         """
-        pass
-
+        if cls.RESOURCE_NAME == 'people':
+            return PeopleQuerySet()
```

Comment:
Me and Sebastian both agreed that there had to be a better way to do this since the abstraction was almost pointless in the super class if we had to specify a case for every subclass.